app/controllers/connections_controller.rb

```diff
@@ -0,0 +1,20 @@
+class ConnectionsController < ApplicationController
```

Comment:
I would still name it as `RelationshipsController` instead. Connection reminds me connections to the database.